### DISCUSSION TITLE
How to add emojies

### DISCUSSION THREAD

**Discussion by giswqs** [giswqs/GEOG510-SP24#2] [Q&A]
Your page looks cool! How did you add emojies?

I use [geemap ](https://geemap.org) to do this

```python
## This is the code
import geemap
m = geemap.Map()
```

![image](https://github.com/giswqs/GEOG510-SP24/assets/5016453/1d849c74-3ba1-4fca-a110-2e572316f313)

**Comment by Andyzxm** [comment]
Watch Dr.Wu's [Youtube ](https://www.youtube.com/c/Qiushengwu?themeRefresh=1)Channel.

**Comment by nathansturgill** [comment]
The hyperlink did not work
